src/css/profile/mobile/common/bottombar.less

```diff
@@ -33,7 +37,7 @@
     }
 }
 
-@media all and (min-width: 685px) and (min-height: 411px) {
+@media all and (min-width: 673px) and (min-height: 411px) {
     .ui-footer .ui-bottom-bar {
         margin: 0 auto;
         width: 75%;
```

Comment:
I think you have to change also `min-width` to 625px, because there are also left/right paddings 2*24px

Reply:
Thanks~
I'll change it.

Reply:
&.ui-bottom-bar-icons {
            min-width: 665 * @px_base;
        }
I need also change 665->645 ?
In case of icon+text , left/right paddings 2*10?

Reply:
yes.

Reply:
but 673px - 2 * 10px = 653px